### PR TITLE
feat(donate): stripe checkout page integration

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

**Review by raisedadead — COMMENTED** [review]
We need to DRY this code.

**Comment by raisedadead** [comment]
@scissorsneedfoodtoo can you review and make recommendations on DRY for this?

**Comment by scissorsneedfoodtoo** [comment]
@raisedadead, did my best to DRY up the code. It's not totally necessary to break the donation function out into its own file, but I thought it looked cleaner this way.

Please let me know if there are any changes you'd like to make.

File: partials/stripe-donation.hbs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            })
+                .then(function (result) {
+                    if (result.error) {
+                        var displayError = document.getElementById('error-message');
+                        displayError.textContent = result.error.message;
+                    }
+                });
+        };
+    })();
+</script>
```
Comment (raisedadead) [inline partials/stripe-donation.hbs]:
```suggestion
</script>

```

New line

**Comment by raisedadead** [comment]
I think this is great.

**Comment by raisedadead** [comment]
Can we test this on .dev and merge? Thanks.

**Comment by scissorsneedfoodtoo** [comment]
Sure thing. I'll add the newline and get this up on .dev.

**Comment by raisedadead** [comment]
> Sure thing. I'll add the newline and get this up on .dev.

I did that already, you can just squash, merge, test on .dev and upload this to .org

**Comment by scissorsneedfoodtoo** [comment]
Great, thank you. I'll merge it now.

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Pull request merged**